### PR TITLE
Feature/profile questions 2

### PR DESCRIPTION
**Pull request by pavlo-chernykh**
what was done:
now it's possible to add more work history fields  (and delete, but minimum 1 field)
image
now it's possible to add more education (and delete, but minimum 1 field)
image
added front-end logic to send data to the server
added routes constant

### CODE REVIEW

File: apps/freelance/src/pages/ProfileQuestions/page1.tsx [hunk 1]
```diff
@@ -122,129 +128,181 @@ export const ProfileQuestions1 = () => {
...
+                        <St.StMinusOutlined onClick={() => remove(name)} />
+                      )}
+                    </Input.Group>
+                  </Form.Item>
+                </St.StFormList>
+              ))}
+              <Form.Item>
+                <Button
+                  type="dashed"
+                  onClick={() => add()}
```
Comment (lyudad) [inline apps/freelance/src/pages/ProfileQuestions/page1.tsx]: onClick={add}

File: apps/freelance/src/redux/services/types/user.types.ts [hunk 2]
```diff
@@ -23,6 +23,10 @@ export interface UpdateUser {
   skills?: number[];
 }
 
+// export interface AddWorkhistoryArr {
+
+// }
```
Comment (lyudad) [inline apps/freelance/src/redux/services/types/user.types.ts]: Снимок экрана 2022-11-21 в 21 20 11

File: apps/freelance/src/pages/ProfileQuestions/page1.tsx (hunk 1 above)
Comment (pavlo-chernykh) [reply apps/freelance/src/pages/ProfileQuestions/page1.tsx]: fixed

File: apps/freelance/src/redux/services/types/user.types.ts (hunk 2 above)
Comment (pavlo-chernykh) [reply apps/freelance/src/redux/services/types/user.types.ts]: fixed

File: apps/freelance/src/pages/ProfileQuestions/page1.tsx [hunk 3]
```diff
@@ -122,129 +128,181 @@ export const ProfileQuestions1 = () => {
...
+                  onClick={add}
+                  block
+                  icon={<PlusOutlined />}
+                >
+                  {t('description.profileQp1.add_one_more')}
+                </Button>
+              </Form.Item>
+            </>
+          )}
+        </Form.List>
```
Comment (lyudad) [inline apps/freelance/src/pages/ProfileQuestions/page1.tsx]: component is toooooooo big, devide

Comment (pavlo-chernykh) [reply apps/freelance/src/pages/ProfileQuestions/page1.tsx]: fixed

**Review by lyudad — APPROVED** [review]

**Pull request merged**